### DISCUSSION TITLE
Please remove illuminate/http from require

### DISCUSSION THREAD

**Discussion by andrewdwallo** [CaashApp/plaid-sdk-laravel#3] [General]
Laravel already has this setup. Whenever I try composer require for your package I can't because composer says:
"- Root composer.json requires caashapp/plaid-sdk-laravel ^0.1.0 -> satisfiable by caashapp/plaid-sdk-laravel[v0.1.0].
    - caashapp/plaid-sdk-laravel v0.1.0 requires illuminate/http ^8.79 -> found illuminate/http[v8.79.0, ..., 8.x-dev] but these were not loaded, likely because it conflicts with another require."

**Comment by Indemnity83** [comment]
Andrew, 

Sounds like you're probably trying to use this library with Laravel 9? I haven't tested it with Laravel 9 yet, which is probably why you're getting this error. You're right that laravel already includes the illuminate/http library but I included it so tests could be run on the library on its own (without having to install it into a Laravel application).

**Reply by andrewdwallo** [reply]
So I need to use this with Laravel 8? is there a way to use it with Laravel 9?
